packages/odyssey-react/src/components/Field/Field.tsx

```diff
@@ -120,10 +125,15 @@ function Label(props: PropsLabel) {
     as = "label",
   } = props;
 
-  const Tag = as;
-
   const label = (
-    <Tag className={styles.label} htmlFor={inputId}>
+    <Box
+      as={as}
+      className={styles.label}
+      htmlFor={inputId}
+      display="flex"
+      alignItems="flex-end"
+      justifyContent="space-between"
+    >
```

Comment:
Is this change required? we're already wrapping `children` in `<Text />` below so text styles should already be inherited for children? Perhaps I'm not understanding the affect of adding Box here entirely can you fill me in?

Reply:
Same question. If we're going to do a replacement here, it feels more natural to do something like `<Text as={as} ...>` and then place our "default" `label` and `legend` styles in `Text.module.scss`.

Reply:
@edburyenegren-okta Your question here is a separate issue from what my feedback above. I'm asking about the need for changing `Tag` to `Box` here because we already wrap the injected `children` with a `Text` node below so they should have "default" text styling in place yes?

To answer your question: at least today, Text doesn't support the polymorphism that Field.Label needs, specifically a `legend` or `label` element. I'm not sure I have a strong opinion on if that should or should not be possible. Moving the "default" label and legend styles into `Text.module.scss` is something I'd need to see a spike PR to evaluate I think.